src/Connection.cpp

```diff
@@ -29,6 +29,7 @@ Connection::Connection(QObject *p, QSslSocket *qtsSock) : QObject(p) {
 	qtsSocket->setParent(this);
 	iPacketLength = -1;
 	bDisconnectedEmitted = false;
+	csCryptPtr.reset(new CryptStateOCB2());
```

Comment:
```suggestion
	csCryptPtr = std::make_unique<CryptStateOCB2>();
```
I think this makes it clear that we're actually assigning this value for the first time instead of replacing it.